Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -708,25 +709,17 @@ class UnrealResourcePreparer
 };
 
 static std::string getCacheDatabaseName() {
-  // On Android, EngineUserDir returns a "fake" directory. UE's IPlatformFile
-  // knows how to resolve it, but we can't pass it to cesium-native because
-  // cesium-native expects a real path. IAndroidPlatformFile::FileRootPath
-  // looks like it should be able to resolve it for us, but that's difficult
-  // to call from here.
-
-  // At the same time, apps on Android are isolated from each other and so we
-  // can't really share a cache between them anyway. So we store the cache in a
-  // different directory on Android.
-#if PLATFORM_ANDROID
-  FString baseDirectory = FPaths::ProjectPersistentDownloadDir();
-#else
-  FString baseDirectory = FPaths::EngineUserDir();
-#endif
-
-  FString filename =
-      FPaths::Combine(baseDirectory, TEXT("cesium-request-cache.sqlite"));
-  UE_LOG(LogCesium, Verbose, TEXT("Caching Cesium requests in %s"), *filename);
-  return TCHAR_TO_UTF8(*filename);
+  FString CesiumFolder =
+      FPaths::Combine(*FPaths::ProjectSavedDir(), TEXT("Cesium"));
+  if (!IFileManager::Get().DirectoryExists(*CesiumFolder)) {
+    IFileManager::Get().MakeDirectory(*CesiumFolder, true);
+  }
+  FString CesiumDBFile =
+      FPaths::Combine(*CesiumFolder, TEXT("cesium-request-cache.sqlite"));
+  FString PlatformAbsolutePath =
+      IFileManager::Get().ConvertToAbsolutePathForExternalAppForWrite(
+          *CesiumDBFile);
+  return TCHAR_TO_UTF8(*PlatformAbsolutePath);
```

Comment:
Previously we stored the cache in a user-specific engine directory everywhere except Android. That meant all your Cesium for Unreal projects could share a cache, which is pretty useful for e.g. Cesium World Terrain. With this change the cache is per-project. We at least need to call that out in CHANGES.md. And we should be sure it's what we want.

Reply:
@kring I'll leave it to you to decide the action here. If we want to maintain status quo and use per-project cache only for mobile devices, then lets keep that (and I think thats the right move). Feel free to push an update to this branch.